src/app/_samples/full-portal/top-app-mashup/top-app-mashup.component.ts

```diff
@@ -242,12 +243,25 @@ export class TopAppMashupComponent implements OnInit {
 
     } );
 
-    this.scservice.selectPortal()
-    .then( () => {
-      const thePortal = this.scservice.getAppPortal();
+    const thePortal = this.scservice.getAppPortal();
+  
+    // Note: myLoadPortal and myLoadDefaultPortal are set when bootstrapWithAuthHeader is invoked
+    if(thePortal){
+      console.log(`Loading specified appPortal: ${thePortal}`);
       window.myLoadPortal("app-root", thePortal, [], null);   // this is defined in bootstrap shell that's been loaded already
-    })
-
+    }else if(window.myLoadDefaultPortal){
+      console.log(`Loading default portal`);
+      window.myLoadDefaultPortal("app-root", [], null);
```

Comment:
I noticed we don't include that null final argument in SDK-R.  Might it be left off for SP-A as well.  I suspect undefined would be treated the same as null within JS Engine for that argument.  (same for myLoadPortal)

Reply:
Yes, null is treated as undefined as they're doing a truthy check. We don't pass in SDK-R...Ok, I'll remove.